internal/project/core_test.go

```diff
@@ -230,3 +231,34 @@ func TestCore_DeleteProject(t *testing.T) {
 		})
 	}
 }
+
+func TestCore_ListKeys(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockRepo := mocks.NewMockIRepo(ctrl)
+	core := NewCore(mockRepo)
+	ctx := context.Background()
+
+	tests := []struct {
+		expected []string
+		wantErr  bool
+		err      error
+	}{
+		{
+			expected: []string{"project_1", "project_2"},
+			wantErr:  false,
+			err:      nil,
+		},
+		{
+			expected: nil,
+			wantErr:  true,
+			err:      fmt.Errorf("Something went wrong"),
```

Comment:
Is this err required because we are not asserting any particular error?

Reply:
if it is not added as part of test struct, then we will need to add conditional statements which is not ideal

Reply:
wantErr is being used for that only , we can assert.Equal(wantErr, err!=nil)

Reply:
The problem is with line no 263. I need to mock the error response. I don't want to have conditional statement on wantErr to return the error response